tests/tests/test_create_multiple_capabilities.py

```diff
@@ -0,0 +1,121 @@
+"""
+API automated test for ACC model app

```

Comment:
Improvise the docstring. 
List all the steps that are happening in the test would be good. Because without saying Creating ACC model, Delete an ACC model doesn't sound right.